colortv/docs/androidTVSDK.md

```diff
@@ -589,34 +601,31 @@ public class MainActivity extends Activity {
 }
 ```
 
-## Customizable Recommendation Center
+## Customization
 
-If you want to customize `Recommendation Center` we have added `ColorTvContentRecommendationConfig` (accessible from `ColorTvRecommendationsController`) which provides methods that allow you to do it.
+There is possibility to customize both Recommendation Center and UpNext layouts. In order to do it, you need to use `ColorTvContentRecommendationConfig`, which provides methods that allow you to change default recommendation layouts into custom ones.
 
-Layouts containing RecyclerView and items are fully customizable. You can change one, both or use the default.
-Process of designing is no different from the usual layout creation. You can use all types of Views or Layouts, there are no limits.
-
-!!! note ""
-    Once the layout is set, it is stored in the singleton config. If you want to reset settings to defaults, invoke the resetToDefault() method.
-
-### ColorTvContentRecommendationConfig
-
-To start customizing recommendation center you have to get `ColorTvContentRecommendationConfig`:
+You can retrieve the config object from `ColorTvRecommendationsController` instance:
 
 ```java
 ColorTvRecommendationsController recommendationsController = ColorTvSdk.getRecommendationsController();
 ColorTvContentRecommendationConfig recommendationConfig = recommendationsController.getConfig();
 ```
 
-Using the config object, you can access the following methods:
+Most of its methods receive `Device` enum type as one of arguments. It is nested inside the `ColorTvContentRecommendationConfig` class. It can be one of: `TV`, `MOBILE` and `TABLET`. It lets you to set various settings for different device types. In order to set custom layouts, you need to provide android layout id leading to aprropriate resource. Process of designing is simillar to the usual layout creation. You can use all types of Views or Layouts, but in order to have injected some recommendations data into them, you need to use some specified ids assigned to proper view types. Although you don not need to use them all and you can add as many additional views as you want.
+
+!!! note ""
+    All settings are stored in the shared preferences, so once they are set in an app they are used in every app launch. In order to restore default layouts you need to invoke `recommendationConfig.resetToDefault()` method.
```

Comment:
Look on implementation, configuration is same only on runtime, that's why it was `singleton`